test/CMakeLists.txt

```
@@ -199,3 +199,9 @@ foreach (_file IN LISTS UNIT_TESTS)
     endif()
 
 endforeach()
+
+# add additional TUs if required
+if (TARGET interop_allocs.pass)
+    target_sources(interop_allocs.pass PRIVATE "pstl_offload/memory/interop_allocs_system.cpp")
+    target_sources(interop_allocs.pass PRIVATE "pstl_offload/memory/interop_allocs_usm.cpp")
```

The minimum supported cmake is 3.11, however `target_sources` can handle relative paths only since 3.13. 
 `${CMAKE_CURRENT_LIST_DIR}` provides the path to the folder with the currently processed `CMakeLists.txt`. 

Also, `target_source` supports multiple items.

```suggestion
    target_sources(interop_allocs.pass PRIVATE 
                   "${CMAKE_CURRENT_LIST_DIR}/pstl_offload/memory/interop_allocs_system.cpp"
                   "${CMAKE_CURRENT_LIST_DIR}/pstl_offload/memory/interop_allocs_usm.cpp")
```

---

Done.